libs/web-components/global-search/src/global-search.scss

```
@@ -0,0 +1,143 @@
+
+                &.fds-global-search-input-focus {
+                    border-width: 1px;
+                }
+
+                .fds-global-search-text-field {
+                    border: none;
+                }
+
+                fds-divider {
+                    visibility: visible;
+                    margin-left: var(--fds-spacing-4) !important;
+                    margin-right: var(--fds-spacing-4) !important;
+                }
+            }
+
+
+            .fds-global-search-container {
+                display: none;
+
+                .fds-global-search-block {
+                    margin-top: var(--fds-spacing-2) !important;
+                    padding-left: var(--fds-spacing-4) !important;
+                    padding-right: var(--fds-spacing-4) !important;
+                    margin-bottom: var(--fds-spacing-2) !important;
+
+                }
+
+                &.fds-global-search-container-open {
+                    display: flex !important;
+                    flex-direction: column;
+                    width: 100%;
+                }
+
+            }
+        }
+
+        fds-linear-progress {
+            width: 100%;
+        }
+    }
+
+}
+
+.fds-global-search {
+    // position: fixed;
+    display: inline-flex;
+    pointer-events: none;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+
+    .fds-global-search-wrapper {
+        width: var(--fds-global-search-width, 400px);
+        justify-content: center;
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        z-index: 1000;
+        border-color: var(--fds-primary, #694ED6);
+        border-width: 1px;
+        border-style: solid;
+        border-radius: var(--fds-spacing-4);
+        pointer-events: auto;
+        @include fds.property(background, background);
+
+
+        .fds-global-search-text-container {
+            width: 100%;
+            height: var(--fds-spacing-5);
+
+            .fds-global-search-text-field {
+                display: flex;
+                flex-direction: row;
+                align-items: center;
+                place-content: space-between;
+                height: 100%;
+                padding-left: var(--fds-spacing-4) !important;
+                padding-right: var(--fds-spacing-4) !important;
+                // border-color: var(--fds-primary, #694ED6);
+                // border-width: 1px;
+                // border-style: solid;
+                // border-radius: var(--fds-spacing-4);
+
+                .fds-global-search-text-input {
+                    height: 100%;
+                    width: 100%;
+
+                    input {
+                        border: none;
+                        width: 100%;
+                        height: 100%;
+                        font-size: calc(2 * var(--fds-spacing-2)) !important;
+                        background: transparent;
+                        color: var(--fds-on-background);
```

Avoid using variables, replace by the fds property 🤓
```suggestion
                        @include fds.text-color(on-background);
```